sympy/sets/tests/test_conditionset.py

```diff
@@ -131,8 +131,11 @@ def test_subs_CondSet():
 
 
 def test_subs_CondSet_tebr():
-    # to eventually be removed
-    c = ConditionSet((x, y), {x + 1, x + y}, S.Reals)
+    with warns_deprecated_sympy():
+        ConditionSet((x, y), {x + 1, x + y}, S.Reals) == \
+            ConditionSet((x, y), Eq(x + 1, 0) & Eq(x + y, 0), S.Reals)
```

Comment:
Seems to be missing `assert`

Reply:
I've just added the assert although this code comment doesn't show as "outdated" for some reason